src/Database/Statement/BufferedStatement.php

```diff
@@ -155,7 +156,7 @@ public function execute(?array $params = null): bool
     public function fetchColumn(int $position)
     {
         $result = $this->fetch(static::FETCH_TYPE_NUM);
-        if (isset($result[$position])) {
+        if ($result && isset($result[$position])) {
```

Comment:
can you make this `$result !== false && ...` ?

Reply:
Done